src/client/dfuse/dfuse_fuseops.c

```diff
@@ -87,7 +87,7 @@ dfuse_fuse_init(void *arg, struct fuse_conn_info *conn)
 	conn->want |= FUSE_CAP_READDIRPLUS;
 	conn->want |= FUSE_CAP_READDIRPLUS_AUTO;
 
-	conn->time_gran = 1000000000;
```

Comment:
This would have been set for a reason, although it probably dates from IOF.  I think it might have been to mask clock-skew between client and server and timestamps apparently going backwards however I will check this in the commit history and report my findings.